torch/csrc/jit/script/init.cpp

```diff
@@ -432,20 +432,8 @@ void initJitScriptBindings(PyObject* module) {
   // public.
   py::class_<Module, std::shared_ptr<Module>>(m, "ScriptModule")
       .def(py::init<>())
-      .def("export", [](const std::shared_ptr<Module> m) {
-        std::string module;
-        RawDataExportMap export_map;
-        std::tie(module, export_map) = ExportModule(m);
-        std::unordered_map<std::string, py::bytes> python_serialized_export_map;
-        for (auto& kv : export_map) {
-          auto t = kv.second;
-          size_t copy_bytes = t.type().elementSizeInBytes() * t.numel();
-          // TODO: this is an unecessary copy. In theory we can directly return
-          // the map from identifier to Tensor, but we need some API in Python
-          // to get raw `bytes` containing the raw tensor data.
-          python_serialized_export_map[kv.first] = py::bytes(static_cast<const char*>(t.data_ptr()), copy_bytes);
-        }
-        return std::make_tuple(py::bytes(module), python_serialized_export_map);
+      .def("export", [](const std::shared_ptr<Module> m, const std::string& filename) {
+        ExportModule(m, filename);
```

Comment:
Also, please add `Module::save` to `Module` (the C++ class), and have it just call `ExportModule`. This is to keep the C++ and Python API consistent.